proto/regen/ecocredit/marketplace/v1/tx.proto

```diff
@@ -195,6 +183,37 @@ message MsgBuyResponse {
   repeated uint64 buy_order_ids = 1;
 }
 
+// MsgBuyDirect is the Msg/BuyDirect request type.
+message MsgBuyDirect {
+
+  // buyer is the address of the credit buyer.
+  string buyer = 1;
+
+  // sell_order_id is the sell order ID against which the buyer is trying
+  // to buy
+  uint64 sell_order_id = 2;
+
+  // quantity is the quantity of credits to buy.
+  string quantity = 3;
+
+  // price_per_credit is the price the buyer is willing to pay per credit.
+  cosmos.base.v1beta1.Coin price_per_credit = 4;
```

Comment:
I think we might actually want to keep this as `bid_price`. The `ask_price` and `bid_price` in both sell and buy orders are also price per credit and changing the name here might cause confusion.

Reply:
i just wasn't sure `bid_price` made much sense in this context, as BuyDirect doesn't really involve bidding on anything. i was thinking of it as similar to going to the grocery store, I wouldn't really "bid" on a bag of chips 🤷🏻

Reply:
ok, switched back https://github.com/regen-network/regen-ledger/pull/967/commits/e1529683ffd0951557abd4d50f1471b4b4acfa73